.packit.yaml

```diff
@@ -0,0 +1,50 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_container_gateway.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-smart_proxy_container_gateway.spec
+    - .packit.yaml
```

Comment:
Can we consistently indent this?

Reply:
Ha probably because it's copy and pasted all over the organization. I'll fix em.